spec/rails_helper.rb

```diff
@@ -46,6 +46,12 @@
   # instead of true.
   config.use_transactional_fixtures = true
 
+  config.before do
+    allow_any_instance_of(StorageRoot) # rubocop:disable RSpec/AnyInstance
+      .to receive(:version_directories)
```

Comment:
It doesn't feel good to have this kind of global stub.  Why do we need it?

Reply:
I wrestled with this a bunch and was unsatisfied with this solution myself. Here's what's up:

The logic in `StorageRoot` for OCFL structures requires there actually being directories on the filesystem, because with OCFL we need the head version in the path which means we need to *find* the head version (unless the logic for the legacy structure which is happy to emit paths that do not exist). Maybe adding the `ocfl-rb` gem will give us a better way to handle this or stub it out in RSpec-land?